File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/koth/PlayerLeftData.java
package com.example.geoquiz_frontend.data.remote.dtos.koth;

public class PlayerLeftData {
    private String lobbyId;

    private String playerId;

    private int totalPlayers;

    public String getLobbyId() { return lobbyId; }
    public String getPlayerId() { return playerId; }
    public int getTotalPlayers() { return totalPlayers; }
}

File: backend/GeoQuiz_backend/GeoQuiz_backend/Application/DTOs/KingOfTheHill/PlayerLeftData.cs
namespace GeoQuiz_backend.Application.DTOs.KingOfTheHill
{
    public class PlayerLeftData
    {
        public Guid LobbyId { get; set; }
        public Guid PlayerId { get; set; }
        public int TotalPlayers { get; set; }
    }
}
